pyquil/api/_compiler.py

```diff
@@ -182,7 +182,7 @@ def __init__(
         quilc_endpoint: Optional[str],
         qpu_compiler_endpoint: Optional[str],
         device: AbstractDevice,
-        timeout: int = 10,
+        timeout: float = 10,
```

Comment:
Should we also support this in `get_qc`, maybe as `compiler_timeout`? That's the "tutorial" way of constructing this in any case.

Reply:
Donezo. I have refrained from adding a timeout option to `QuantumComputer#compile`, because (1) I think the options available now are sufficient, and (2) we could probably find a million places to add a timeout flag to (gotta draw the line somewhere).

Reply:
agreed